stdlib/LinearAlgebra/src/eigen.jl

```diff
@@ -9,25 +9,22 @@ matrix `A`. This is the return type of [`eigen`](@ref), the corresponding matrix
 factorization function.
 
 If `F::Eigen` is the factorization object, the eigenvalues can be obtained via
-`F.values` and the eigenvectors as the columns of the matrix `F.vectors`.
+`F.values`, the eigenvectors and left eigenvectors as the columns of the matrices
+`F.vectors` and `F.vectorsl`, and error bounds for eigenvalues and eigenvectors
+`F.eerrbd` and `F.verrbd`.
+
 (The `k`th eigenvector can be obtained from the slice `F.vectors[:, k]`.)
 
-Iterating the decomposition produces the components `F.values` and `F.vectors`.
+Iterating the decomposition produces the components `F.values`, `F.vectors`, `F.vectorsl`,
+`F.eerrbd`, `F.verrbd`.
```

Comment:
Doesn't this mean that values, vectors = eigen will not work anymore?

Reply:
No - the lhs tuple needs not to be exhaustive. You can even do `a, = [1,2,3]` to set `a` to `1`.